pkgs/development/tools/dump_syms/default.nix

```
@@ -17,8 +17,13 @@
 let
   pname = "dump_syms";
   version = "2.2.2";
+
+  # Workaround for https://github.com/nixos/nixpkgs/issues/166205
+  workaround = lib.optionalAttrs stdenv.cc.isClang {
```

```suggestion
  env = lib.optionalAttrs stdenv.cc.isClang {
```

Then put `env` in the expression below.

---

TIL about `env`, that works for me!